M2/Macaulay2/e/f4/varpower-monomial.hpp

```diff
@@ -1,91 +1,17 @@
 /* Copyright 2006 by Michael E. Stillman */
+#pragma once
```

Comment:
> @d-torrance is this the case also for all of your compilations?

Yup!  It looks like `#pragma once` has been supported in gcc since version 3.4, and the oldest gcc I care about is the one in Ubuntu 18.04, which uses gcc 7.

Reply:
Great! I think we should use them more widely.